spec/integration/bisect_spec.rb

```diff
@@ -33,14 +33,19 @@ def bisect(cli_args, expected_status=nil)
       end
     end
 
-    context "when the bisect commasaturingnd is long" do
```

Comment:
😆

Reply:
I mean I think this is what its supposed to say :joy:

Reply:
Ahah

Reply:
it's probably vim magic 🧙🏻‍♂️